autoload/LanguageClient.vim

```diff
@@ -860,4 +860,50 @@ function! LanguageClient#cquery_vars(...) abort
     return call('LanguageClient#find_locations', ['$cquery/vars'] + a:000)
 endfunction
 
+function! LanguageClient_contextMenuItems() abort
+  return {
```

Comment:
Please keep the indent consistent, i.e., 4 spaces.

Reply:
Fixed.